tracer/src/Datadog.Trace/TraceContext.cs

```diff
@@ -203,6 +204,16 @@ public void CloseSpan(Span span)
                     _spans = default;
                     TelemetryFactory.Metrics.RecordCountTraceSegmentsClosed();
                 }
+                else if (CIVisibility.IsRunning && span.Type is SpanTypes.TestSession or SpanTypes.TestModule or SpanTypes.TestSuite or SpanTypes.Test or SpanTypes.Browser)
```

Comment:
nit: do you think this should be an extension method somewhere? I can imagine it coming up again...

```csharp
internal static class SpanExtensions
{
    public bool IsCiVisibilitySpan(this ISpan span)
        => span.Type is SpanTypes.TestSession or SpanTypes.TestModule or SpanTypes.TestSuite or SpanTypes.Test or SpanTypes.Browser;
}
```

```suggestion
                else if (CIVisibility.IsRunning && span.IsCiVisibilitySpan())
```

Reply:
done in [7609a88](https://github.com/DataDog/dd-trace-dotnet/pull/5549/commits/7609a88f288a9dd6ff6ab26f4a7675458ab21bab)